### DISCUSSION TITLE
Beta version updates through discover app on steam deck

### DISCUSSION THREAD

**Discussion by burninater69** [unknownskl/xbox-xcloud-client#982] [Q&A]
I am using the 2.0 beta releases on steam deck and was curious if there is a way to update greenlight through the discover app rather than downloading the appimage every time a new beta version is released.

**Comment by gunnaraas** [comment]
Making installation easier for Steam Deck users, who may be unfamiliar with installing and maintaining an `.appimage`, is a great idea. 

Personally, I would like to see this available to install directly from the Steam Store. The most user-friendly experience is to avoid sending the user into desktop mode entirely.

**Comment by Gangialf90** [comment]
I fully agree on the question, it would be much more comfortable to have the ability to update the application directly from the discover store of Steam Deck Desktop Mode; it would be even nicer to have the ability to update it through the Steam system but maybe you ask too much!

> I am using the 2.0 beta releases on steam deck and was curious if there is a way to update greenlight through the discover app rather than downloading the appimage every time a new beta version is released.

**Comment by unknownskl** [comment]
It is possible to download Greenlight via the discovery store already. Flatpak support was added since 2.0.1 or something like that
